docs/debugging/errors/exception-bus-bus-unavailable.rst

```diff
@@ -28,7 +28,7 @@
     Exception: bus.Bus unavailable
 
 Error above means you haven't configured :doc:`longpolling <../../admin/about_longpolling>` properly. Longpolling is
-used for instant notifications and updates. If you are sure that you don't need
+used for instant notifications and updates when multiple workers is enabled i.e number of workers set in the configuration file is greather than one. If you are sure that you don't need
```

Comment:
@ajepe actually "is greater ``or equal`` one", i.e. when workers have a non-zero value

Reply:
Yes you are right, I always assumed one or more.